### PR TITLE
Updated RAMClustR tool

### LINKED ISSUE #392: RAMClustR: Add filtering, qc and plots for related steps
Add the options to save plots generated with RAMClustR in a collection of PDFs

The filtering part creates the MSData output - the filtering section should also be made available in the wrapper with 2 options:
* blank
* cv
 both of them being a boolean and both can be selected.

If blank is selected, the parameters for the function need to be added (qc.tag, blank.tag, signal_to_noise (sn) and remove.blanks (boolean)).
None means that the filtering is disabled.
CV requires a `qc.tag` and a `cv` value (coefficient of variation).
The order of execution is filtering blanks < normalization < filtering cv.

The filtering step creates the following output
Image

The rc.qc. function should be added - this should be added as a boolean.
It is executed as last step of the workflow.
The rc.qc function creates the following output :
Image

### PR DESCRIPTION
**Pull request by zargham-ahmad**
This PR includes the following:

- Added filter blanks, filter cv, qc as part of the tool.
- Produce additional graph output if filter cv or qc is selected.
- added test data and test for this.

_I have added the test data for the new test which is present used in the package itself as it contain qc and blanc sample_ 

_note
The CI should pass after https://github.com/RECETOX/galaxytools/pull/504 gets merged_

closes #392

### CODE REVIEW

File: tools/ramclustr/macros.xml [hunk 1]
```diff
@@ -184,9 +184,55 @@
...
+                    help="Extractor for xcms objects in preparation for clustering. This function offers normalization 
+                    by total extracted ion signal. It is recommended to first run 'Filter blanks' to remove non-sample derived signal">
+                    <option value="FALSE" selected="true">FALSE</option>
+                    <option value="TRUE">TRUE</option>
+                </param>
+                <when value="TRUE">
+                    <param label="qc tag" name="qc_tag" type="text" value="QC"
+                        help="Character vector of length one or two. If length is two, enter search string and factor name in $phenoData slot 
+                        (i.e. c('QC', 'sample.type'). If length one (i.e. 'QC'), will search for this string in the 'sample.names' slot by default.
+                        Default is 'QC'"/>
```
Comment (hechth) [inline tools/ramclustr/macros.xml:230]: Could be maybe moved to a token but I'm not sure if nested macros work.

File: tools/ramclustr/ramclustr.xml
Comment (hechth) [inline tools/ramclustr/ramclustr.xml:1]: Galaxy wrapper version needs a bump :).

**Review by hechth — COMMENTED** [review]
Galaxy wrapper version needs a bump - otherwise seems fine :)

File: tools/ramclustr/ramclustr.xml
Comment (zargham-ahmad) [reply tools/ramclustr/ramclustr.xml:1]: done in [db6c005](https://github.com/RECETOX/galaxytools/pull/514/commits/db6c00547f65dddbb37559d22301085379f642b5)

**Review by hechth — APPROVED** [review]

**Pull request merged**